### PR TITLE
Example to print IMU data

### PR DESCRIPTION
**Pull request by dr0ne-guy**
I have created a python file that prints the IMU data from the FC

### CODE REVIEW

File: examples/highres_imu.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+        print(f"Z Acceleration (m/s^2): {imu.acceleration_frd.down_m_s2}")
+        print(f"X Gyro (rad/s): {imu.angular_velocity_frd.forward_rad_s}")
+        print(f"Y Gyro (rad/s): {imu.angular_velocity_frd.right_rad_s}")
+        print(f"Z Gyro (rad/s): {imu.angular_velocity_frd.down_rad_s}")
+        print(f"X Mag (gauss): {imu.magnetic_field_frd.forward_gauss}")
+        print(f"Y Mag (gauss): {imu.magnetic_field_frd.right_gauss}")
+        print(f"Z Mag (gauss): {imu.magnetic_field_frd.down_gauss}")
+        print(f"Temperature (°C): {imu.temperature_degc}")
+        print("-----------------------------------------")
+        await asyncio.sleep(0)  # Sleep for a second between each print
```
Comment (JonasVautherin) [inline examples/highres_imu.py:38]: This is not sleeping for a second, and in any case you should not sleep here :see_no_evil:

**Review by JonasVautherin — CHANGES_REQUESTED** [review]

**Comment by dr0ne-guy** [comment]
@JonasVautherin I have changed the sleep timer.

**Comment by JonasVautherin** [comment]
But you should not sleep there :see_no_evil:. You should consume the events as they arrive. Sleeping may just slow them down and accumulate a delay that you don't want :innocent:.

**Comment by dr0ne-guy** [comment]
I have removed the delay 😅

**Review by julianoes — APPROVED** [review]
Thanks for that!

**Comment by julianoes** [comment]
If you can fix the style issue then this is good to go in, thanks.
https://github.com/mavlink/MAVSDK-Python/actions/runs/10682857899/job/29632558957?pr=727

**Comment by dr0ne-guy** [comment]
I have changed the style

**Pull request merged**